molecule/aap-test/verify.yml

```diff
@@ -0,0 +1,88 @@
+- name: Verify
+  hosts: worker_group
+  gather_facts: true
+  become: true
+  vars:
+    test_dir_path: "{{ lookup('env', 'MOLECULE_SCENARIO_DIRECTORY') }}/../../vm-testing/test"
+  vars_files:
+    - vars/vars.yml
+  tasks:
```

Comment:
Most of this file seems to be C&P from https://github.com/securesign/artifact-signer-ansible/blob/main/molecule/default/verify.yml - can we pull the shared tasks out to a common shared file?